crux-dev/dev/tmt/hash_map.clj

```diff
@@ -0,0 +1,91 @@
+(ns tmt.hash_map
```

Comment:
I assume this likely doesn't want to be in the PR, and in it's place something could be put within query test? (Could very well take the test from the branch on my fork)

Reply:
yep, agreed, let's get this into something like `(ns crux.map-keys-test-362)` and have it run as part of the usual test suite, if it's reasonably quick to run?

Reply:
or, if you wouldn't consider it worth keeping, yeah, let's remove it - your call :smile:

Reply:
I was just using this file to do quick tests to identify the bug better. I can remove it if you want but i just kept it there because it gives a more detailed view of the bug, I'll write an explicit test for the bug as well though.

Reply:
After talking with dan i'm going to remove it and just write a test